packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2755,7 +2715,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 content,
                 serializedContent,
                 maxOpSize,
-                this._flushMode === FlushMode.TurnBased,
+                this._flushMode === FlushMode.TurnBased || this._orderSequentiallyCalls > 0,
```

Comment:
The code above where it adds `batch: true` to ops also needs to be updated.
It may be simpler to update the `flushMode` to `TurnBased` in `orderSequentially` if its `Immediate` (similar to what is happening today). We can still update it internally without exposing any APIs to do it.